frontend/src/components/Register.css

```diff
@@ -0,0 +1,38 @@
+body{
```

Comment:
Don't give this background as body as It will impact whole project body.

Reply:
but how to apply background to entire page.